articles/logout/index.md

```diff
@@ -123,6 +123,8 @@ In order to avoid validation errors, make sure that you do include the protocol
 
 * The `returnTo` parameter does not function for all social providers. Please check your social provider's settings to ensure that they will accept the `redirectTo` parameter.
 
+* The URLs provided to the **Allowed Logout URLs** list are case-sensitive, so the URL used for logouts must match the case of the logout URL configured on the dashboard. Note, that the scheme and host parts, are case-insensitive. For example, in `http://www.Example.Com/FooHoo.html`, the `http://www.Example.Com` is case-insensitive, while the `FooHoo.html` is case-sensitive.
```

Comment:
@erjohnson You missed the clarification by Yvonne about the scheme and host parts, I updated the PR